src/css/tools/_font.scss

```diff
@@ -0,0 +1,4 @@
+@mixin font ($size){
+  font-size: $size;
+  font-family: 'Lato', sans-serif;
```

Comment:
Everytime you use this mixin it will add `font-family` to that rule which seems unnecessary. If only one font-familiy is used then the rule should exist only once in the css.